src/main/java/cloud/fogbow/ras/core/plugins/authorization/SuperUserAwareAuthorizationPlugin.java

```diff
@@ -26,8 +26,7 @@
     public SuperUserAwareAuthorizationPlugin() {
         classFactory = new ClassFactory();
         superUserOnlyOperations = new ArrayList<String>();
-        String superUserOperationsListString = PropertiesHolder.getInstance().getProperty(ConfigurationPropertyKeys.SUPERUSER_OPERATIONS_KEY);
-        for (String superUserOperationString : superUserOperationsListString.split(SystemConstants.OPERATION_NAMES_SEPARATOR)) {
+        for (String superUserOperationString : SystemConstants.SUPERUSER_ONLY_OPERATIONS.split(SystemConstants.OPERATION_NAMES_SEPARATOR)) {
```

Comment:
This is not a command on this line, but on something else that could be changed in the constructor. Instead of having its own ClassFactory, could you use cloud.fogbow.ras.core.AuthorizationPluginInstantiator.getAuthorizationPlugin(className)? Also, could you create the default plugin at the constructor, instead of creating it everytime isAuthorized is called? However, in this case, I am not sure whether the recursive call to AuthorizationPluginInstantiator.getAuthorizationPlugin would work.

Reply:
Done, I think it is better now.